src/vroom_errors.h

```diff
@@ -89,7 +89,7 @@ class vroom_errors {
       cpp11::sexp warn_call = Rf_lang3(
           warn,
           Rf_mkString(
-              "One or more parsing issues, see `problems()` for details"),
+              "One or more parsing issues, use `problems()` on your data frame for details"),
```

Comment:
What do you think of "call" (instead of "use")?

I wonder if we should give a concrete demo?

```
One or more parsing issues, call `problems()` on your data frame for details, e.g.:
  dat <- vroom(...)
  problems(dat)
```

It always feels awkward to emit rich messages from C++, so let's try to decide on the content without worrying too much about the mechanics at first.

Reply:
I think `call` is definitely better and an example would be nice and explicit